src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -536,6 +536,7 @@ internal override void ForceComplete(SourceLocation? locationOpt, CancellationTo
                         break;
 
                     case CompletionPart.Members:
+                        // PROTOTYPE(semi-auto-props): Can we add initializers here?
```

Comment:
>Can we add initializers here? [](http://example.com/codeflow?start=55&length=29)

Could you elaborate please?

Reply:
@AlekseyTs Currently, adding initializers causes a cycle. The comment suggests adding the initializers at this point instead. See the other PROTOTYPE comment in the same file.